helm/aws-load-balancer-controller/values.yaml

```diff
@@ -26,6 +26,12 @@ serviceAccount:
   name:
   # Automount API credentials for a Service Account.
   automountServiceAccountToken: true
+  # List of image pull secrets to add to the Service Account.
+  # e.g.
+  # imagePullSecrets
+  # - name: docker
+  imagePullSecrets:
```

Comment:
How about this for the comment and the value:
```
# imagePullSecrets specifies the list of image pull secrets to add to the ServiceAccount
imagePullSecrets:
    # - name: docker
```

Reply:
Updated!

Reply:
Thanks